include/openssl/ssl.h

```diff
@@ -4501,6 +4506,30 @@ OPENSSL_EXPORT int SSL_CTX_set_max_send_fragment(SSL_CTX *ctx,
 OPENSSL_EXPORT int SSL_set_max_send_fragment(SSL *ssl,
                                              size_t max_send_fragment);
 
+
+
+// SSL_CTX_set_post_handshake_auth and SSL_set_post_handshake_auth enable the
+// Post-Handshake Authentication extension (pha_ext) to be added to the ClientHello
+// such that post-handshake authentication can be requested by the server.
+// If val is 0 then the extension is not sent, if it is 1 then it is.
+// By default, the extension is not sent. On the client side, a certificate callback
+// will need to be set via SSL_CTX_set_client_cert_cb() if no certificate is
```

Comment:
Nit: Just our typical coding style
```suggestion
// will need to be set via |SSL_CTX_set_client_cert_cb| if no certificate is
```